packages/sitecore-jss-react/src/components/BYOCComponent.tsx

```diff
@@ -122,12 +122,13 @@ export class BYOCComponent extends React.Component<BYOCComponentProps> {
       },
       errorOverride: 'BYOC: This component was not registered.',
     };
-
-    const fallbackComponent = this.props.missingComponentComponent ? (
-      <this.props.missingComponentComponent {...unRegisteredComponentProps} />
-    ) : (
-      <MissingComponent {...unRegisteredComponentProps} />
-    );
+    // For now we'll make fallback client-only - to ensure client-only BYOC components won't cause 'missing' error frame in SSR
+    const fallbackComponent =
+      typeof window !== 'undefined' && this.props.missingComponentComponent ? (
```

Comment:
Seems like you'd still see `<MissingComponent/>` in this case? 
Regardless, wonder if this should be functionality in `FEAAS.ExternalComponent` not here?

Reply:
@Inviz what are your thoughts?

Reply:
Discussed, figured out 👍